src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,30 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+interface Props {
+  selectedTodo: Todo | null;
```

Comment:
```suggestion
  selectedTodo: Todo;
```
`selectedTodo` should be required, it doesn't make sense to render modal without it